pavjacko/renative
# Use same version of RNV within platformTemplates

Due to things, when a spawned rnv process operated on files within `/platformTemplates` it would be the version as defined in the `package.json` of the target package. It was therefore difficult to change rnv and be confident your changes were apparent without publishing to npm first.

This PR checks the current process and if the rnv package has been `yarn/npm link`'d it will preload the require.cache with the linked package and ignore the package.json version. This will only occur for projects linked and not more "standard" use cases 

To test:
 - checkout `renative` repo and run `yarn run link`
 - in another project, buildable by rnv (e.g. sdk-ui) 
 - amend an export from rnv and see it become available (without publishing) in `/packages/app/platformBuilds/_shared/configs.js` (perhaps during `rnv run -p web`).
